platform/vs/docker-sonic-vs/start.sh

```diff
@@ -180,6 +180,8 @@ supervisorctl start tunnelmgrd
 
 supervisorctl start fabricmgrd
 
+supervisorctl start rebootbackend
```

Comment:
>rebootbackend [](http://example.com/codeflow?start=20&length=13)

I could not find any code relating this program. And the name is weird.

Reply:
rebootbackend is the main process inside framework container handling all processing of reboot commands.